server/models/submissionSchema.model.js

```diff
@@ -62,20 +62,19 @@ const submissionSchema = new Schema(
   //   removedFromJudging: Boolean,
   // },
   {
-    isSubmitted: { type: Boolean, default: false },
+    isSubmitted: { state: String },
     attributes: {
       title: { type: String, default: '' },
       description: { type: String, default: '' },
       technologies: { type: [String], default: [] },
     },
     urls: { type: [{ _id: false, label: String, url: String }], default: [] },
     categories: {
-      type: [{ _id: false, categoryID: String, categoryName: String }],
-      default: [],
+      _id: String,
     },
     flags: { type: [String], default: [] },
     numFlags: { type: Number, default: 0 },
-    removedFromJudging: { type: Boolean, default: false },
+    removedFromJudging: { visible: Boolean },
```

Comment:
same issue as `isSubmitted` above. `removedFromJudging` should be renamed to `visible`. There's no need to give it sub-json